heat/core/tests/test_factories.py

```diff
@@ -2,6 +2,7 @@
 import torch
 
 import heat as ht
+from torch._C import Value
```

Comment:
Why is this here? is it used?

Reply:
No. It was automatically added by the editor, but it is not used.